content/en/docs/architecture/images.md

```diff
@@ -0,0 +1,132 @@
+As noticed, the github organization `openshift-priv` is used. Every commits merged to any repo of `openshift` is merged to the counterpart repo of `openshift-priv` as well. The role of `openshift-priv` is out of the scope of this page.
+{{< /alert >}}
+
+The above snippet defines
+* where to get `Dockerfile` of the build: `Dockerfile.rhel7` in `release-{MAJOR}.{MINOR}` branch of repo cloned from `git@github.com:openshift-priv/cluster-etcd-operator.git`. The branch name `release-{MAJOR}.{MINOR}` is resolved by the branch name of the `cluster-etcd-operator.yml` file. In the example, it is `release-4.6`.
+* how to replace the images in `From` directives of `Dockerfile.rhel7`: `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6` is replaced by `golang` and `registry.ci.openshift.org/ocp/4.6:base` is by `openshift-enterprise-base`. 
+
+## CI Images VS. ART Images
+From the above introduction, for each component, the CI image and the ART image are built from the same `Dockerfile` (DPTP has automation to refresh the `Dockerfile` path and name in the `ci-operator`'s configuration if either of them is modified in `ocp-build-data`). They are the same image if the images in `From` directives of `Dockerfile` are replaced by the same images. Actually, it is the case. In the above example, 
+ART **ensures** that `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6` is the same as `golang` in the ART build and `openshift-enterprise-base` is also an OCP component. Moreover, building the image from `Dockerfile` on a developer's computer leads to the same image as well.
+
+## GoLang Version
+As in the above example, the OCP component `cluster-etcd-operator` uses `GoLang-1.15` to build the image. When the version is evolved, ART team will modify the `GoLang` version in `ocp-build-data`, e.g., `GoLang-1.16`. Then, DPTP automation will create pull requests when it detects that `Dockerfile` in the component repo uses a different `GoLang` version from what is expected by `ocp-build-data`:
+
+* a PR (`PR1`) to the component repo: modify `From` directives of `Dockerfile`. It is up to the owner of the component whether or not to merge the PR.
+
+* a PR (`PR2`) to the `openshift/release` repo: modify the `ci-operator`'s configuration. This PR is merged automatically.
+
+{{< alert title="Info" color="info" >}}
+To avoid a large number of jobs triggered at the same time from PRs created from OCP components up to `GoLang` version change, the automation creates `PR1`s in sequence. It usually takes 1 or 2 days to complete this process.
+{{< /alert >}}
+
+Depending on when `PR1`s are merged, ART could use different versions of `GoLang` to build images.
+
+* Before code freeze: if less than half of the components merge `PR1`s, ART uses the old version `GoLang-1.15`. Otherwise, the new version `GoLang-1.16` for every component that has merged `PR1` and the old version `GoLang-1.15` for every component that has not.
```

Comment:
Let's just prefer sentences and paragraphs - not everything needs to be bullet list.